packages/gatsby-source-drupal/src/gatsby-node.js

```diff
@@ -267,7 +286,25 @@ exports.sourceNodes = async (
           return data
         }
 
-        const data = await getNext(url)
+        let data = []
+        if (isTranslatable === false) {
+          data = await getNext(url)
+        } else {
+          for (let i = 0; i < languageConfig.enabledLanguages.length; i++) {
+            let currentLanguage = languageConfig.enabledLanguages[i]
+            let dataForLanguage
+
+            if (currentLanguage === languageConfig.defaultLanguage) {
+              dataForLanguage = await getNext(url)
+            } else {
+              const baseUrlWithoutTrailingSlash = baseUrl.replace(/\/$/, ``)
+              const urlPath = url.href.replace(`${baseUrlWithoutTrailingSlash}/${apiBase}/`, ``)
+              dataForLanguage = await getNext(`${baseUrlWithoutTrailingSlash}/${currentLanguage}/${apiBase}/${urlPath}`)
```

Comment:
I'm open to handling this in a different way or using a library for the URL parsing. I don't like having to manually edit the URL and it's error prone.